SPECS/abseil-cpp/abseil-cpp.spec

```diff
@@ -11,12 +14,19 @@ Source0:        https://github.com/abseil/abseil-cpp/archive/refs/tags/%{version
 BuildRequires:  cmake >= 3.20.0
 BuildRequires:  gcc
 BuildRequires:  make
-
-%if 0%{?with_check}
-BuildRequires:  gmock >= 1.12.0
-BuildRequires:  gmock-devel >= 1.12.0
-BuildRequires:  gtest >= 1.12.0
-BuildRequires:  gtest-devel >= 1.12.0
+BuildRequires:  ninja-build
+BuildRequires:  gcc-c++
+
+BuildRequires:  gmock
+BuildRequires:  gmock-devel
+BuildRequires:  gtest
+BuildRequires:  gtest-devel
+
+%ifarch s390x
```

Comment:
Can delete this section if you want,

Reply:
Can drop the gtest if you are talking about these dependencies:
Could NOT find GTest (missing: GTEST_LIBRARY GTEST_INCLUDE_DIR"